src/main/java/com/exactpro/epfast/template/xml/ByteVectorFieldXml.java

```diff
@@ -0,0 +1,25 @@
+package com.exactpro.epfast.template.xml;
+
+import com.exactpro.epfast.template.ByteVectorField;
+
+import javax.xml.bind.annotation.XmlAttribute;
+
+public class ByteVectorFieldXml extends FieldInstrContent implements ByteVectorField, InstructionXml {
+
+    private ApplicationIdentity lengthFieldId = new ApplicationIdentity(null);
```

Comment:
You should always pass something to constructor. And I guess this something will be `this` in 100% of cases.